src/create-json-ad.md

```diff
@@ -0,0 +1,102 @@
+
+[JSON-AD](core/json-ad.md) is the default serialization format of Atomic Data.
+It's just JSON, but with some extra requirements.
+
+Most notably, all keys are links to [Atomic Properties](https://atomicdata.dev/classes/Property).
+These Properties must be actually hosted somewhere on the web, so other people can visit them to read more about them.
+
+Ideally, in JSON-AD, each Resource has its own `@id`.
+This is the URL of the resource.
+This means that if someone visits that `@id`, they should get the resource they are requesting.
+That's great for people re-using your data, but as a data provider, implementing this can be a bit of a hassle.
+That's why there is a different way that allows you to create Atomic Data _without manually hosting every resource_.
+
+## Creating JSON-AD without hosting individual resources yourself
+
+In this section, we'll create a single JSON-AD file containing various resources.
+This file can then be published, shared and stored like any other.
+
+The goal of this preparation, is to ultimately import it somewhere else.
+We'll be importing it to Atomic-Server.
+Atomic-Server will create URLs for every single resource upon importing it.
+This way, we only deal with the JSON-AD and the data structure, and we let Atomic-Server take care of hosting the data.
+
+Let's create a BlogPost.
+We know the fields that we need: a `name` and some `body`.
+But we can't use these keys in Atomic Data, we should use URLs that point to Properties.
+We can either create new Properties (see the Atomic-Server tutorial), or we can use existing ones, for example by searching on [AtomicData.dev/properties](https://atomicdata.dev/properties).
+
+## Using Ontologies, Classes and Ontologies
+
+Ontologies are groups of concepts that describe some domain.
+For example, we could have an Ontology for BlogPosts or .
+
+At this moment, there are few models created in Atomic Data.
+
+## Setting the first values
+
+```json
+{
+  "https://atomicdata.dev/properties/name": "Writing my first blogpost",
+  "https://atomicdata.dev/properties/description": "Hi! I'm a blogpost. I'm also machine readable!",
+}
+```
+
+## Multiple items
+
+If we want to have _multiple_ items, we can simply use a JSON Array at the root, like so:
+
+```json
+[{
+  "https://atomicdata.dev/properties/name": "Writing my first blogpost",
+  "https://atomicdata.dev/properties/description": "Hi! I'm a blogpost. I'm also machine readable!",
+},{
+  "https://atomicdata.dev/properties/name": "Another blogpost",
+  "https://atomicdata.dev/properties/description": "I'm writing so much my hands hurt.",
+}]
+```
+
+## Preventing deduplication with `localId`
+
+When we want to _publish_ Atomic Data, we also want someone else to be able to _import_ it.
+An important thing to prevent, is _data duplication_.
+If you're importing a list of Blog posts, for example, you'd want to only import every article _once_.
+
+The way to prevent this, is by adding a `localId`.
```

Comment:
Too vague, missing, or wrong back-reference: The way to prevent what exactly?

Better to spell it out: The way to prevent *duplication* is...

Reply:
Agreed!